.circleci/config.yml

```diff
@@ -404,6 +404,46 @@ jobs:
           no_output_time: "3m0s"
           <<: *check_scion_connections
 
+      - run:
+          name: NEW -- Check updated configuration matches VPN setup
+          working_directory: ~/project/.circleci
+          command: |
+            # Check configuration on AP
+            echo "Client AP 1405 new VPN configuration"
+            docker-compose exec -T as1405 /bin/bash -c \
+              "apt-get install -y jq > /dev/null 2>&1; "\
```

Comment:
Could you move the installation of `jq` to the Dockerfile

Reply:
No, since the host containers do (intentionally) not get rebuilt, that has no effect until the next update